bluemira/equilibria/fem_fixed_boundary/utilities.py

```diff
@@ -0,0 +1,90 @@
+# bluemira is an integrated inter-disciplinary design tool for future fusion
+# reactors. It incorporates several modules, some of which rely on other
+# codes, to carry out a range of typical conceptual fusion reactor design
+# activities.
+#
+# Copyright (C) 2021 M. Coleman, J. Cook, F. Franza, I.A. Maione, S. McIntosh, J. Morris,
+#                    D. Short
+#
+# bluemira is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# bluemira is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with bluemira; if not, see <https://www.gnu.org/licenses/>.
+
+"""Module to support the fem_fixed_boundary implementation"""
+
+import math
+
+import dolfin
+import numpy as np
+
+
+def b_coil_axis(r, z, pz, curr):
+    """
+    Return the module of the magnetic field of a coil (of radius r and centered in
+    (0, z)) calculated on a point on the coil axis at a distance pz from the
+    axis origin.
+    """
+    return 4 * math.pi * 1e-7 * curr * r**2 / (r**2 + (pz - z) ** 2) ** 1.5 / 2.0
```

Comment:
There's `np.pi` which could be used here, then we don't need to import `math`

Reply:
modified.